Update the using-a-git-provider-access-token.adoc to omit scm-userid (#2587)
codenvy/che-docs: modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

<1> Your {prod-short} user ID.
<2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
<3> The Git provider URL.

> <4> The Git provider username or user's email address for `azure-devops`.

Reviewer: ```suggestion
<4> The Git provider username. For `azure-devops`, the user's email address.
```

> <4> The Git provider username, user email in case you are using `azure-devops`.

Reviewer: ```suggestion
<4> The Git provider username or user's email address for `azure-devops`.
<4> The Git provider username. For `azure-devops`, the user's email address.
```

<1> Your {prod-short} user ID.
<2> The Git provider name: `github` or `gitlab` or `bitbucket-server` or `azure-devops`.
<3> The Git provider URL.
<4> The Git provider username or user's email address for `azure-devops`.

> <4> The Git provider username. For `azure-devops`, the user's email address.

Author: ```suggestion
```
